src/lint/linter/ArcanistGoFmtLinter.php

```diff
@@ -27,9 +27,17 @@ public function getLinterConfigurationName() {
   }
 
   public function getBinary() {
+    $go_imports = $this->getGoImportsBinary();
+    if (Filesystem::binaryExists($go_imports)) {
+      return $go_imports;
+    }
     return 'gofmt';
   }
 
+  public function getGoImportsBinary() {
+    return 'goimports';
```

Comment:
Just curious why a separate public function just for returning a string literal?